Dockerfile.jdk8

```diff
@@ -1,5 +1,12 @@
 # This image will be published as dspace/dspace
 # See https://dspace-labs.github.io/DSpace-Docker-Images/ for usage details
+# 
+# This version is JDK8 compatible
+# - tomcat:8
```

Comment:
As this is the `jdk8` Docker file, should this be using `tomcat:8-jre8`?  I'm not sure what JRE/JDK is in use by the default `tomcat:8` image.